### DISCUSSION TITLE
Qual a situação atual desse projeto?

### DISCUSSION THREAD

**Discussion by victorsapateiro** [YuriAlessandro/gerador-ficha-tormenta20#380] [Perguntas e Respostas]
Opa, tudo bom?

Só queria saber qual a situação atual do projeto. Confesso que a possibilidade de criar fichas customizadas é muito interessante especialmente na falta de um app oficial atualizado.

Outra coisa, já pensaram em abrir um Patreon ou algo semelhante pra quem quiser apoiar vocês?

**Comment by YuriAlessandro** [comment]
Olá. Nem um pouco de demora em responder. Mas isso já indica a situação atual do projeto, eu acho?

Não trabalhamos nele em 2024, essa é a verdade. Eu ainda estou atualizando o mapa e a Caverna do Saber de tempo em tempo, mas o gerador de ficha está parado. Nós temos um beta, com algumas classes, algumas raças e quase nenhum poder e quase nenhuma magia.

A verdade é que dá trabalho. Muito trabalho. Não é só copiar os textos para o código. É escrever código que automatiza cada poder, cada magia.

Acabou que a ideia ficou pelo caminho, não-finalizada.

Sobre abrir um patreon ou algo do tipo, ainda não passa pela nossa cabeça também. Nós não temos tanto tempo para dedicar ao projeto, e isso só geraria expectativas de mais atualizações e mais trabalho, eu acho.

O que eu pretendo fazer é atualizar o gerador aleatório para a versão mais recente, e acho que vai ser isso por um tempo.

**Reply by victorsapateiro** [reply]
poxa cara, que pena. Se tiver como ajudar da um toque... tenho acesso a todo o material e sei codar ok

**Reply by airtonosorio** [reply]
pode avisar aqui tb, eu posso tentar ajudar

**Comment by YuriAlessandro** [comment]
Mais vivo do que nunca agora, com a nova update.
